### PR TITLE
Ziua7

### PR DESCRIPTION
**Pull request by Wiuwiti**

### CODE REVIEW

File: src/views/evaluation/Evaluation.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
-const insertHeader = function(option = {}){
+const InsertHeader = function(option = {}){
     const result = []
     for(var i = 0;i < option.length; i++){
```
Comment (gcazaciuc) [inline src/views/evaluation/Evaluation.js]: .map

File: src/views/evaluation/Evaluation.js [hunk 2]
```diff
@@ -37,27 +37,28 @@ const tableRows = function(option = {}){
...
-const tableConstructor = function (option ={} , Header ={} ){
+const TableConstructor = function (option ={} , Header ={} ){
     const result = []
-    result.push(tableHeader(Header))
-    for (var i = 0; i < option.length; i++){
-         result.push(tableRows(option[i]))
-    }
+    result.push(TableHeader(Header))
+    const result2 = option.map(function(element){
+        return `${TableRows(element)}`
```
Comment (gcazaciuc) [inline src/views/evaluation/Evaluation.js]: no need for template string. Replace with arrow function

File: src/views/evaluation/Evaluation.js [hunk 3]
```diff
@@ -37,27 +37,28 @@ const tableRows = function(option = {}){
...
 
-const tableConstructor = function (option ={} , Header ={} ){
+const TableConstructor = function (option ={} , Header ={} ){
     const result = []
-    result.push(tableHeader(Header))
-    for (var i = 0; i < option.length; i++){
-         result.push(tableRows(option[i]))
-    }
+    result.push(TableHeader(Header))
+    const result2 = option.map(function(element){
```
Comment (gcazaciuc) [inline src/views/evaluation/Evaluation.js]: result and result2 are lousy names. find better, more descriptive ones

File: src/views/new-evaluation/newEvaluation.js [hunk 4]
```diff
@@ -1,92 +1,96 @@
...
-        ${result.join('')}
+            ${result.join('')}
         <li id="extra"></li>
         <div class="clearfix"></div>
     </ul>`
 }
 
-const technicalLevelPickerBottom = function (option = {}){
-    const result = []
+const TechnicalLevelPickerBottomSection = function (option = {}){
```
Comment (gcazaciuc) [inline src/views/new-evaluation/newEvaluation.js:38]: param should be called options

Comment (gcazaciuc) [reply src/views/new-evaluation/newEvaluation.js:38]: because they can be multiple options

File: src/views/new-evaluation/newEvaluation.js [hunk 5]
```diff
@@ -1,92 +1,96 @@
...
-    for (var i = 0; i < option.length; i++){
-        for( var j = 0; j < option[i].inputLevel.length; j++){
-            result.push(`<li class = "${option[i].classes[j]}" id= "${option[i].ID[j]}">
-            <input type="radio" name="level" value = "${option[i].inputLevel[j]}">
-        </li>`)
-        }
-    }
+
+const TechnicalLevelPickerBottom = function (option = {}){
+    let result = option.map(function(element){return TechnicalLevelPickerBottomSection(element)})
```
Comment (gcazaciuc) [inline src/views/new-evaluation/newEvaluation.js]: use an arrow fn here

File: src/views/new-evaluation/newEvaluation.js [hunk 6]
```diff
@@ -1,92 +1,96 @@
...
+const TextAreaBox = function(option = {}){
     return `<div class="user-textarea">
         <h3>${option.headerTitle}</h3>
         <textarea placeholder="${option.placeHolder}"required></textarea>
     </div>`
 }
-const textArea = function (option = {}){
+const TextArea = function (option = {}){
     const result = []
     for(var i = 0 ; i < option.length; i++){
```
Comment (gcazaciuc) [inline src/views/new-evaluation/newEvaluation.js]: map

File: src/views/new-evaluation/newEvaluation.js [hunk 7]
```diff
@@ -96,29 +100,29 @@ const selectConstructor = function(option= {}){
...
         </li>
         `)
     }
     return result.join('')
 }
 
-const newEvaluationForm = function (option = {}){
+const NewEvaluationForm = function (option = {}){
     var result = []
     for(var i = 0; i < option.length; i++){
```
Comment (gcazaciuc) [inline src/views/new-evaluation/newEvaluation.js]: map

**Review by gcazaciuc — APPROVED** [review]

**Review by gcazaciuc — APPROVED** [review]

File: src/views/new-evaluation/newEvaluation.js (hunk 7 above)
Comment (Wiuwiti) [reply src/views/new-evaluation/newEvaluation.js]: Unfortunately I have been outside almost all night long and didn't managed to change the code asap as I promised during this day of work :).
Hopefully the new version I got ATM is good enough 👍 used everywhere I could maps and arrow keys to make them as short and as readable as possible

**Pull request merged**